src/main/scala/beam/sim/BeamHelper.scala

```diff
@@ -667,10 +674,65 @@ trait BeamHelper extends LazyLogging {
     * @param outputDirectory output folder where full configs will be generated
     */
   private def writeFullConfigs(config: TypesafeConfig, outputDirectory: String): Unit = {
-    val configConciseWithoutJson = config.root().render(ConfigRenderOptions.concise().setFormatted(true).setJson(false))
+
+    val configResolveOptions = ConfigResolveOptions.defaults().setAllowUnresolved(true)
```

Comment:
Could you, please, extract this to separate abstraction? `BeamHelper` keeps growing...

Reply:
Move config writing in saperate class `ConfigHelper` that handle writing operation